modules/nw-creating-a-route.adoc

```diff
@@ -66,15 +65,15 @@ kind: Route
 metadata:
   name: hello-openshift
 spec:
-  host: hello-openshift-hello-openshift.<Ingress_Domain> <1>
+  host: www.example.com <1>
   port:
     targetPort: 8080 <2>
   to:
     kind: Service
     name: hello-openshift
 ----
-<1> `<Ingress_Domain>` is the default ingress domain name. The `ingresses.config/cluster` object is created during the installation and cannot be changed. If you want to specify a different domain, you can specify an alternative cluster domain using the `appsDomain` option.
-<2> `targetPort` is the target port on pods that is selected by the service that this route points to.
+<1> The `host` field is an alias/DNS that points to the service. It can be any valid DNS name, such as `www.example.com`. If not specified, a route name is automatically generated. This field must follow DNS952 subdomain conventions.
```

Comment:
may be better
The `host` field is an alias/DNS name that points to the service, It can be any valid DNS name, such as `www.example.com`. but this field must follow DNS952 subdomain conventions. If not specified, a route name is automatically generated.